src/main/java/com/wepay/zktools/zookeeper/internal/ZooKeeperClientImpl.java

```diff
@@ -55,6 +58,14 @@
     private volatile boolean running = true;
 
     public ZooKeeperClientImpl(String connectString, int sessionTimeout) throws ZooKeeperClientException {
+        this(connectString, sessionTimeout, null);
+    }
+
+    /**
```

Comment:
Nit: Do we want to java doc the inputs so the user knows what sessionTimeout and connectTimeout are for?

Reply:
Yes, we should update javadoc.

Reply:
Updated javadoc, and also change TimeUnit.SECONDS to TimeUnit.MILLISECONDS to be in accordance with *sessionTimeout* unit. PTAL @smithakoduri